.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,40 @@
-## What changes were proposed in this pull request?
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
```

Comment:
[Here](https://spark.apache.org/contributing.html) it's not mentioned that the PR title should reflect the solution and not the issue. It's not template related but would be good to mention somewhere because it's a common pattern.